tests/validation-time/Test.LexemeFL.fst

```diff
@@ -0,0 +1,13 @@
+///  A new lexeme __FL__ has been added to show file and line (file(line)) to make writing tests easier.
+/// This file is line sensitive any edit will change the value of __FL__.
+module Test.LexemeFL
+open FStar.Tactics.V2
+open FStar.String
+module LT = FStar.List.Tot
+// Kinda funky to get a good validation time test, added Strings in other PR will fix this.
+// The lexer is sending back some strange character that we have to adjust.
+let fl = __FL__ 
+let _ = assert(fl <> "")
+let fl'  = string_of_list (list_of_string "Test.LexemFL.fst(11)")
+let _ = assert((strlen fl') = 20) by compute()
+let _ = assert(fl' = "Test.LexemFL.fst(11)") by compute()
```

Comment:
Using `assert_norm` breaks the dependency on tactics (and is maybe more idiomatic).

Reply:
Also what is this testing? Seems to be just that 1) `fl` is nonempty and 2) that string_of_list and list_of_string are inverses.
(Note the missing 'e' from 'Test.LexemFL.fst(11)'.)

Reply:
No what the darn string_of_list (list_of_string is doing is getting rid of oddball characters that prevent line 13 from working to show that __FL__ is working.

Reply:
I have a commit pushed with the assert_norm changes. Do I need another PR?

Reply:
So I have Test.FLLexeme moved to the directory to be renamed later, micro-benchmarks and validaton-time deleted and out of the makefile. 

And this FStar revision will compile Luke's fl(), which is two characters shorter and more obvious than __FL__. I have no idea that reflection would do this for me as it is not in the book. 

It would obviate this entire PR.

 Guido which way do you want it? Lexeme or Tactics?